### PR TITLE
feat: bump to webpack@5.61.0

### PR DESCRIPTION
**Pull request by jesec**
Bug: #805, webpack/webpack#14532

### CODE REVIEW

**Review by styfle — APPROVED** [review]
Thanks!

File: test/watcher.test.js [hunk 1]
```diff
@@ -101,7 +101,6 @@ it('Should support custom watch API', async () => {
   await new Promise((resolve, reject) => {
     const watcher = new CustomWatchFileSystem(function watchStart (files, dirs, missing) {
       expect(files._set.size).toBeGreaterThan(100);
-      expect(missing._set.size).toBeGreaterThan(40);
```
Comment (styfle) [inline test/watcher.test.js:104]: Why was `missing` removed here?

Comment (jesec) [reply test/watcher.test.js:104]: I found it unlikely to be useful.

As seen in earlier commit, https://github.com/vercel/ncc/commit/1bd2aac221b2e53ce2fc0a7c7f79425518310526#diff-b886328ab4b91304e8f0ff17323f05d868c3f83cbb0a6c492f362fdedcd70f96, the line was changed to accommodate the new version of webpack.

Comment (styfle) [reply test/watcher.test.js:104]: @guybedford Thoughts on this? I'm fine with removing it.

**Pull request merged**